.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "sgl-kernel/3rdparty/cutlass"]
```

Comment:
May we specify the latest tag v3.6.0

Reply:
It seems tag cannot be specified. But I have specified the commit.

Reply:
I see